libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/DataType.scala

```diff
@@ -44,6 +44,8 @@ object DataType {
   case class UpdateId(uuid: UUID) extends UUIDKey
   object UpdateId extends UUIDKeyObj[UpdateId]
 
+  final case class SourceUpdateId(get: String) extends AnyVal
```

Comment:
Could we use `value` instead of `get` to mirror (most of the) other value classes?

Reply:
Sure, I will change that, thanks for the tip.

IMO it would be helpful to have a code style guide where all these kinds of things are written down. It would help keep our codebase consistent. @simao @koshelev